index.html

```diff
@@ -4325,106 +4319,66 @@ <h2>Acknowledgements</h2>
 community facilitated by Christopher Allen, Shannon Appelcline, Kiara Robles,
 Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny, Drummond Reed, Joe
 Andrieu, Heather Vescent, Kim Hamilton Duffy, Samantha Chase, Andrew Hughes,
-Erica Connell, Shigeya Suzuki, and Zaïda Rivai. The participants in the Internet
-Identity Workshop, facilitated by Phil Windley, Kaliya Young, Doc Searls, and
-Heidi Nobantu Saul, also supported the refinement of this work through numerous
-working sessions designed to educate about, debate on, and improve this
-specification.
+Will Abramson, Erica Connell and Eric Schuh. The participants in the
+Internet Identity Workshop, facilitated by Phil Windley, Kaliya Young, Doc
+Searls, and Heidi Nobantu Saul, also supported the refinement of this work
+through numerous working sessions designed to educate about, debate on, and
+improve this specification.
       </p>
 
       <p>
-The Working Group also thanks our Chair, Brent Zundel, our ex-Chair Kristina
-Yasuda, as well as our W3C Staff Contact, Ivan Herman, for their expert
+The Working Group also thanks our Working Group Chair Brent Zundel, and ex-chair
+Kristina Yasuda, as well as our W3C Staff Contact, Ivan Herman, for their expert
 management and steady guidance of the group through the W3C standardization
-process.
+cycle. We also thank the Chairs of the W3C Credentials Community Group,
+Christopher Allen, Joe Andrieu, Kim Hamilton Duffy, Heather Vescent, Wayne
+Chang, Mike Prorock, Harrison Tang, Kimberly Wilson Linson, and Will Abramson,
+who oversaw the incubation of this work.
       </p>
 
       <p>
 Portions of the work on this specification have been funded by the United States
 Department of Homeland Security's Science and Technology Directorate under
 contracts 70RSAT20T00000029, 70RSAT21T00000016, 70RSAT23T00000005,
 70RSAT20T00000010/P00001, 70RSAT20T00000029, 70RSAT21T00000016/P00001,
-70RSAT23T00000005, 70RSAT23C00000030, 70RSAT23R00000006, 70RSAT24T00000011, and the National
-Science Foundation through NSF 22-572. The content of this specification does
-not necessarily reflect the position or the policy of the U.S. Government and no
-official endorsement should be inferred.
+70RSAT23T00000005, 70RSAT23C00000030, 70RSAT23R00000006, 70RSAT24T00000011, and
+the National Science Foundation through NSF 22-572. The content of this
+specification does not necessarily reflect the position or the policy of the
+U.S. Government and no official endorsement should be inferred.
       </p>
 
       <p>
 The Working Group would like to thank the following individuals for reviewing
-and providing feedback on the specification (in alphabetical order):
+and providing feedback on and implementations of the specification
+(in alphabetical order):
```

Comment:
It says "in alphabetical order", but the list below isn't. A fix cannot be suggested over the full range of the list (as it includes multiple segments of drop and insert). I can make several suggestions that add up to the whole fix, if that would be helpful.

Reply:
I fixed some of the ordering by last name, let me know if that fixes everything you saw, @TallTed.